jenkins/raspberrypi/devops.service

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=Azure DevOps Agent
+After=multi-user.target
+
+[Service]
+# Client Env Vars
+# /etc/devops-environment is a file containing the necessary credentials for running E2E Tests 
+EnvironmentFile=/etc/devops-environment
+
+User=pi
+Type=idle
+ExecStart=/bin/bash /home/pi/myagent/run.sh > /home/pi/myagent/run.log 2>&1
```

Comment:
I think your script doesn't install the agent, so you might want to add a comment with a link to the DevOps agent install steps to get this path - /home/pi/myagent

Reply:
my script installs the devops agent but I've added more explicit comments in the setup_pi_for_e2e_tests.sh